src/test/java/uk/gov/companieshouse/api/accounts/validation/CurrentPeriodValidatorTest.java

```diff
@@ -163,6 +163,39 @@ void validateBalanceSheetWithOtherLiabilitiesOrAssetsTotalNetAssetsError() {
                         ErrorType.VALIDATION.getType())));
     }
 
+    @Test
+    @DisplayName("ERROR - Mandatory elements missing for NetCurrentAssets and TotalAssetsLessCurrentLiabilities")
```

Comment:
Could we add tests for the scenario where either `CurrentAssets` or `OtherLiabilitiesOrAssets` is null while the other is not to improve unit test coverage?